keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -17,17 +17,14 @@
 #include "dynamis.h"
 
 #ifdef ENCODER_ENABLE
-bool encoder_update_kb(uint8_t index, bool clockwise) {
-    if (!encoder_update_user(index, clockwise)) {
-        return false; 
+    bool encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
Please revert this change. 

This is absolutely not the way to handle this, and will cause issues.  Eg, if VIA is not enabled, this will simply not function.

Additionally, Encoder support should NOT be added to the keymap/layout macro, especially not at the keyboard level.  (VIA implementation should not bleed into keyboard code, but needs to be kept in the via keymap, when possible)

Reply:
Ok. I reverted the code and modified that this code run only when via is enabled.